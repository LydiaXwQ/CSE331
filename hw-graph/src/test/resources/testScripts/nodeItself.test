# Tests for a graph with 1 node and 1 edge (point to itself).
# Test that a graph can be created, 1 node and 1 self edge can be added, and the
# nodes and edges can be printed out.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add an edge
AddEdge graph1 n1 n1 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1