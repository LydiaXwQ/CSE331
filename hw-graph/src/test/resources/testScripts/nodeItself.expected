# Tests for a graph with 1 node and 1 edge (point to itself).
# Test that a graph can be created, 1 node and 1 self edge can be added, and the
# nodes and edges can be printed out.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add an edge
added edge e1 from n1 to n1 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: n1(e1)